plugins/catalog/src/components/CatalogTable/CursorPaginatedCatalogTable.tsx

```diff
@@ -39,9 +39,9 @@ export function CursorPaginatedCatalogTable(props: PaginatedCatalogTableProps) {
       columns={columns}
       data={data}
       options={{
+        paginationPosition: 'both',
         ...options,
```

Comment:
I think it makes more sense to put this at the bottom of the list so that all of them can be overridden, but it seemed like it was intentional based on the comment below this line.

Reply:
yep, correct. But your change makes sense to me!